lib/openapi_first/request_validation/request_body_validator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module OpenapiFirst
+  module RequestValidation
+    class RequestBodyValidator
+      def initialize(operation)
+        @operation = operation
+      end
+
+      def validate!(parsed_request_body, request_content_type)
+        request_body = operation.request_body
```

Comment:
this is interesting... what if the body is not defined in the OAS but it's sent in the request? 🤔

Reply:
In that case request validation should pass, because you don't say that request body must be empty. I'll add a test about that in case there is none already.